openAI.js

```diff
@@ -1,11 +1,7 @@
 import OpenAI from "openai";
 import fs from "fs/promises";
 
-import { getOpenAiKey } from "./config.js";
-
-const openai = new OpenAI({
-  apiKey: getOpenAiKey().openaiKey,
-});
+const openai = new OpenAI({ apiKey: process.env.OPEN_AI_KEY });
```

Comment:
If it's a global install then how will it reference the config?

Reply:
it will check the env. but weren't you using process.env to pass in the openai key before this anyway?

Reply:
Nope, maybe merge both with `{...}`

But no need to work on this atm. Let's talk about the next project. We may steal things from here. But I'm not sure this is the correct way to look at scaffolding.